web/src/main/java/com/microsoft/applicationinsights/web/internal/correlation/TraceContextCorrelation.java

```diff
@@ -397,9 +399,12 @@ public static String generateChildDependencyTraceparent() {
             }
 
             RequestTelemetry requestTelemetry = context.getHttpRequestTelemetry();
+            String traceId = requestTelemetry.getContext().getOperation().getId();
+
+            // get the necessary part as traceId in context is of legacy AI format.
+            traceId = traceId.substring(1, traceId.length() - 1);
```

Comment:
you don't need it

Reply:
I needed it before as I was appending the "|". Now I don't. Will update.